Automattic/jetpack
# Connection Screen: make VoiceOver (screen reader) announce lists as such

Fixes https://github.com/Automattic/jetpack/issues/35485

## Proposed changes:

The screen reader VoiceOver (on Safari) doesn't take into account list semantics when `list-style-type` is set to `none` in CSS. However, the lists in the Connect screen look like actual lists and should be identified as such. This PR adds the `role=list` attribute on these lists to fix this.

### Other information:


## Jetpack product discussion


Issue: https://github.com/Automattic/jetpack/issues/35485
Project thread: pf5801-xS-p2

## Does this pull request change what data or activity we track or use?


n/a

## Testing instructions:






Testing is optional since these changes have no impact other than on VoiceOver.

If you want to go the extra mile:
- Spin up a test site with this branch
- Visit these 3 pages and check nothing broke:
  - `/wp-admin/admin.php?page=jetpack#/`
  - `/wp-admin/admin.php?page=jetpack#/connect-user`
  - `/wp-admin/admin.php?page=my-jetpack#/connection`
- Using VoiceOver on Safari, check that lists are announced
<img width="500" alt="Screenshot 2024-02-16 at 1 38 32 PM" src="https://github.com/Automattic/jetpack/assets/1620183/097a71a0-dbaf-4028-b2b7-75ffbcb4b350">
